### DISCUSSION TITLE
How to use trivy after installing it with helm chart to k8s cluster?

### DISCUSSION THREAD

**Discussion by cscareerkaizen** [aquasecurity/trivy#1849] [Q&A]
I know trivy binary is in trivy pod, but when I execute a command like this, it doesn't do anything
```sh
$ kubectl exec -it trivy-0 -- sh
~ $ trivy
NAME:
   trivy - A simple and comprehensive vulnerability scanner for containers

USAGE:
   trivy [global options] command [command options] target

VERSION:
   0.24.0

COMMANDS:
   image, i          scan an image
   filesystem, fs    scan local filesystem for language-specific dependencies and config files
   rootfs            scan rootfs
   repository, repo  scan remote repository
   client, c         client mode
   server, s         server mode
   config, conf      scan config files
   plugin, p         manage plugins
   help, h           Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --quiet, -q        suppress progress bar and log output (default: false) [$TRIVY_QUIET]
   --debug, -d        debug mode (default: false) [$TRIVY_DEBUG]
   --cache-dir value  cache directory (default: "/home/scanner/.cache/trivy") [$TRIVY_CACHE_DIR]
   --help, -h         show help (default: false)
   --version, -v      print the version (default: false)

~ $ trivy -d image nginx 
2022-03-17T15:53:27.666Z        DEBUG   Severities: UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL. # <------ gets stuck here
```

Isn't the use case for trivy k8s pod something like this?
```sh
kubectl exec -it trivy-0 -- trivy image nginx
```

Why doesn't trivy command make any progress?

**Comment by houshym** [comment]
in Kubernetes cluster. there is a trivy service that you can access. of course, I installed Trivy on k8s but I am getting the following error:
curl -v -k http://172.28.12.135:4954
*   Trying 172.28.12.135:4954
* Connected to 172.28.12.135 (172.28.12.135) port 4954(#0)
> GET /ready HTTP/1.1
> Host: 172.28.12.135:4954
> User-Agent: curl/7.77.0
> Accept: */*
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 404 Not Found
< Content-Type: text/plain; charset=utf-8
< X-Content-Type-Options: nosniff
< Date: Fri, 18 Mar 2022 01:02:48 GMT
< Content-Length: 19
<
404 page not found
* Connection #0 to host 172.28.12.135 left intact




of course curl -v -k http://172.28.12.135:4954/healthz
> GET /healthz HTTP/1.1
> Host: 172.28.12.135:4954
> User-Agent: curl/7.77.0
> Accept: */*
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< Date: Fri, 18 Mar 2022 01:02:53 GMT
< Content-Length: 2
< Content-Type: text/plain; charset=utf-8
<
* Connection #0 to host 172.28.12.135 left intact
ok%
